client/my-sites/themes/theme-showcase.jsx

```diff
@@ -664,7 +662,7 @@ class ThemeShowcase extends Component {
 				/>
 				{ this.renderSiteAssemblerSelectorModal() }
 				{ isLoggedIn && (
-					<ThemeShowcaseSurvey condition={ () => lastNonEditorRoute.includes( 'theme/' ) } />
+					<ThemeShowcaseSurvey condition={ () => window.location.pathname.includes( 'theme/' ) } />
```

Comment:
With this condition, the survey will never be visible, since the `pathname` is `/themes/{siteUrl}`.; the `Survey` component is rendered independently of this, but will `return null` if the condition is falsy.

You need to add `previousPath: getPreviousPath( state )` on `mapStateToProps` and then use it for the condition.

Reply:
Addressed in 5203a4e17cf169b997b463a34e08fa06b3e466e6